winml/lib/Api/LearningModelDevice.cpp

```diff
@@ -3,8 +3,12 @@
 
 #include "pch.h"
 #include "LearningModelDevice.h"
+
+#ifdef USE_DML
 #include "core/providers/dml/DmlExecutionProvider/inc/MLOperatorAuthor.h"
 #include "core/providers/dml/DmlExecutionProvider/inc/DmlExecutionProvider.h"
+#endif USE_DML
```

Comment:
can these just delete?

Reply:
yup

---
In reply to: [349806135](https://github.com/microsoft/onnxruntime/pull/2466#discussion_r349806135) [](ancestors = 349806135)